multimedia/x265/Portfile

```diff
@@ -30,9 +30,9 @@ long_description    x265 is a free software library and application for \
                     compression format, and is released under the terms of the \
                     GNU GPL.
 homepage            https://www.videolan.org/developers/x265.html
-master_sites        https://download.videolan.org/pub/videolan/x265/
-distname            ${name}_${version}
-worksrcdir          ${distname}/source
+master_sites        https://github.com/videolan/x265/archive/
+distname            ${version}
```

Comment:
Please use the github portgroup instead of setting these values manually.

Reply:
Thanks for your examples.  I made this change and also removed the livecheck since that's built into the GitHub PortGroup.